functions/src/api/v1/rooms/getRooms.ts

```diff
@@ -0,0 +1,20 @@
+import { Request, Response } from 'express'
+
+/**
+ * @swagger
+ * /v1/rooms/:
+ *   get:
+ *     description: List created rooms.
+ *     tags: ['rooms']
+ *     responses:
+ *       200:
+ *         description: All the rooms linked to this token
+ */
+export const getRooms = async (req: Request, res: Response) => {
+    try {
+        res.send('Hello world')
+    } catch (error) {
+        console.error(error) // eslint-disable-line no-console
+        res.status(500).end()
```

Comment:
why not to use `res.sendStatus` for consistency here ?

Reply:
this is a dumb implementation, this will change when implementing it